Unable to build Lisk Core docker image

### Expected behavior

Should be able to create the Lisk Core docker image.

### Actual behavior

Creating the Lisk Core image fails with the following error:

```
sameer@Sameers-MBP lisk-core % make build
docker build -f ./docker/Dockerfile --build-arg NODEJS_VERSION=18.16.1 --tag=lisk/core .
[+] Building 18.6s (12/14)                                                                                                                                                                           
 => [internal] load build definition from Dockerfile                                                                                                                                            0.0s
 => => transferring dockerfile: 946B                                                                                                                                                            0.0s
 => [internal] load .dockerignore                                                                                                                                                               0.0s
 => => transferring context: 2B                                                                                                                                                                 0.0s
 => [internal] load metadata for docker.io/library/node:18.16.1-alpine                                                                                                                          0.0s
 => [builder 1/5] FROM docker.io/library/node:18.16.1-alpine                                                                                                                                    0.0s
 => [internal] load build context                                                                                                                                                               4.0s
 => => transferring context: 224.72MB                                                                                                                                                           3.8s
 => CACHED [builder 2/5] RUN apk add --no-cache alpine-sdk=~1 python3=~3 libtool=~2 autoconf=~2 automake=~1 rust=~1 cargo=~1 cmake=~3 clang15-libs=~15 clang15-dev=~15 clang15=~15 rustfmt=~1   0.0s
 => CACHED [builder 3/5] WORKDIR /home/lisk/build                                                                                                                                               0.0s
 => CACHED [stage-1 2/6] RUN adduser -D lisk &&     mkdir /home/lisk/.lisk &&     chown -R lisk:lisk /home/lisk/ &&     npm install -g npm                                                      0.0s
 => CACHED [stage-1 3/6] WORKDIR /home/lisk                                                                                                                                                     0.0s
 => [stage-1 4/6] COPY --chown=lisk:lisk ./ .                                                                                                                                                   3.6s
 => [builder 4/5] COPY ./package-lock.json ./package.json ./.npmrc ./                                                                                                                           0.4s
 => ERROR [builder 5/5] RUN npm ci                                                                                                                                                             14.1s
------                                                                                                                                                                                               
 > [builder 5/5] RUN npm ci:                                                                                                                                                                         
#0 3.384 npm WARN deprecated uuid@3.2.1: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.                                                                                                                                                                      
#0 4.076 npm WARN deprecated readdir-scoped-modules@1.1.0: This functionality has been moved to @npmcli/fs                                                                                           
#0 4.250 npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.                                                       
#0 6.631 npm WARN deprecated cli-ux@5.6.7: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
#0 7.973 npm WARN deprecated @oclif/screen@3.0.4: Deprecated in favor of @oclif/core
#0 8.190 npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
#0 10.93 npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
#0 11.61 npm WARN deprecated @oclif/screen@1.0.4: Deprecated in favor of @oclif/core
#0 13.78 npm ERR! path /home/lisk/build/node_modules/@chainsafe/blst
#0 13.79 npm ERR! command failed
#0 13.79 npm ERR! signal SIGSEGV
#0 13.79 npm ERR! command sh -c node dist/scripts/install.js
#0 13.79 
#0 13.79 npm ERR! A complete log of this run can be found in: /home/lisk/.npm/_logs/2023-07-03T15_01_00_411Z-debug-0.log
------
Dockerfile:17
--------------------
  15 |     
  16 |     COPY ./package-lock.json ./package.json ./.npmrc ./
  17 | >>> RUN npm ci
  18 |     
  19 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c npm ci" did not complete successfully: exit code: 1
make: *** [build-image] Error 1
```

### Steps to reproduce

- Checkout Lisk Core v4.0.0-beta.2
- Run `make build-image`

### Which version(s) does this affect? (Environment, OS, etc...)

Lisk SDK v6.0.0-beta.4

### Additional Information

Upgrading `@chainsafe/blst` to `0.2.9` might resolve the issue. Currently, `0.2.6` is in use.

--- a/elements/lisk-cryptography/package.json
+++ b/elements/lisk-cryptography/package.json
@@ -42,7 +42,7 @@
 		"varuint-bitcoin": "1.1.2"
 	},
 	"peerDependencies": {
-		"@chainsafe/blst": "0.2.6",
+		"@chainsafe/blst": "0.2.9",
 		"sodium-native": "3.2.1"
 	},
 	"peerDependenciesMeta": {

--- a/framework/package.json
+++ b/framework/package.json
@@ -41,7 +41,7 @@
 		"test:functional": "jest --config=./test/functional/jest.config.js --runInBand"
 	},
 	"dependencies": {
-		"@chainsafe/blst": "0.2.6",
+		"@chainsafe/blst": "0.2.9",
 		"@liskhq/lisk-api-client": "^6.0.0-beta.4",
 		"@liskhq/lisk-chain": "^0.4.0-beta.4",
 		"@liskhq/lisk-codec": "^0.3.0-beta.3",

